### ISSUE TITLE
[xBind] Add support for x:Bind Hot Reload

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#3242]
##  What would you like to be added:

Add support for x:Bind properties and functions hot reload.

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by MartinZikmund** [comment]
Additional info from #2709:

I think the problem is [here](https://github.com/unoplatform/uno/blob/c39d532b45af3638bc3e800f988c04d56e60fa9e/src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.cs#L85):

```
foreach (var instance in EnumerateInstances(Window.Current.Content, uri))
{
   switch (instance)
   {
#if __IOS__
	case UserControl userControl:
		userControl.Content = XamlReader.Load(fileReload.Content) as UIKit.UIView;
		break;
#endif
        case ContentControl content:
                content.Content = XamlReader.Load(fileReload.Content);
                break;
    }
}
```

The current behavior seems that the reloaded element is actually wrapped as a content of its original instance - `Page` is wrapped inside a `Page` for example.

 This "wrapping issue" was originally noticed by @VitezslavImrysek, and I think it causes this problem too. He sent me the following to show what exactly happens after the reload:

![image](https://user-images.githubusercontent.com/1075116/75615765-adceb700-5b48-11ea-9d89-4368e4699ae7.png)

**Comment by jeromelaban** [comment]
Fixed in Uno 4.0
